implementations/rust/ockam/ockam_command/src/tcp/outlet/create.rs

```diff
@@ -80,28 +88,38 @@ impl CreateCommand {
 
         let is_finished: Mutex<bool> = Mutex::new(false);
 
+        // If no alias is supplied, then create a random one (with a friendly name).
+        let alias = match &self.alias {
+            None => random_name().to_string(),
+            Some(alias) if alias.is_empty() => random_name().to_string(),
+            Some(alias) => alias.clone(),
+        };
```

Comment:
We can avoid this block here removing the `Option` and using clap attributes:

```
#[arg(long, display_order = 901, id = "ALIAS", value_parser = alias_parser, default_value_t = random_name(), hide_default_value = true)]
pub alias: String,
```

Reply:
great catch; done